planning/behavior_path_planner/include/behavior_path_planner/behavior_path_planner_node.hpp

```diff
@@ -99,6 +101,8 @@ class BehaviorPathPlannerNode : public rclcpp::Node
   rclcpp::Publisher<Path>::SharedPtr path_candidate_publisher_;
   rclcpp::Publisher<TurnIndicatorsCommand>::SharedPtr turn_signal_publisher_;
   rclcpp::Publisher<HazardLightsCommand>::SharedPtr hazard_signal_publisher_;
+  rclcpp::Publisher<Marker>::SharedPtr left_bound_publisher_;
+  rclcpp::Publisher<Marker>::SharedPtr right_bound_publisher_;
```

Comment:
Can I change it as a MarkerArray with left and right?
It would be simpler and easier to display.

Reply:
done